src/utils/format.ts

```diff
@@ -13,3 +13,22 @@ export const formatter = (maximum:boolean) => new Intl.NumberFormat('en-US', {
   // (this suffices for whole numbers, but will print 2500.10 as $2,500.1)
   // maximumFractionDigits: 0, // (causes 2500.99 to be printed as $2,501)
```

Comment:
@shan8851 remove ?

Reply:
![Screenshot 2022-07-01 at 07 56 10](https://user-images.githubusercontent.com/39712238/176841047-9dd50f81-7064-4097-a07e-d3d6f18dc87f.png)

@goshatravin I think this was your commit I think a couple of months back. I can remove if not needed though, just let me know.

Reply:
@shan8851 ok leave it